src/Symfony/Component/Notifier/Exception/UnsupportedSchemeException.php

```diff
@@ -144,6 +144,10 @@ class UnsupportedSchemeException extends LogicException
             'class' => Bridge\SmsBiuras\SmsBiurasTransportFactory::class,
             'package' => 'symfony/sms-biuras-notifier',
         ],
+        'smsc' => [
+            'class' => Bridge\Smsc\SmscTransportFactory::class,
+            'package' => 'symfony/smsc-notifier',
```

Comment:
Please add a test case in `UnsupportedSchemeExceptionTest`, thanks

Reply:
Done